src/WhatToCook.WebApp/ClientApp/src/app/meal-planner/meal-plan-creator/meal-plan-creator.component.html

```diff
@@ -0,0 +1,36 @@
+//TODO: do delete button for meal plans
```

Comment:
Are those TODO's relevant? Or is it just for later?

Reply:
irrelevant TODO's are deleted, will be in the next commit